ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -989,8 +994,9 @@ protected void maybeProcessCallbacks() {
               blobInfo = new BlobInfo(serverBlobProperties,
                   decryptCallbackResultInfo.result.getDecryptedUserMetadata().array());
             }
-            int contentSize = decryptCallbackResultInfo.result.getDecryptedBlobContent().remaining();
-            if (!resolveRange(contentSize)) {
+            totalSize = decryptCallbackResultInfo.result.getDecryptedBlobContent().remaining();
+            chunkSize = totalSize;
```

Comment:
there was a small issue with how `chunkSize` and `totalSize` was assigned for encrypted simple blobs before, but in practice it did not matter since the `chunkSize` set would always be larger than the decrypted chunk size, so the math in `filterChunkToRange` worked out.